doc/getting_started/setup_verilator.md

```diff
@@ -228,15 +220,17 @@ The output consists of a textual "waveform" representing the SPI signals.
 
 ## Generating waveforms (optional)
 
+TODO(lowRISC/opentitan[#13042](https://github.com/lowRISC/opentitan/issues/13042)): the below does not work with the Bazel test running flow.
```

Comment:
thank you for adding a note here to the issue!

Reply:
in case i missed it above...should we mention anything about where the artifacts are so we can perform the steps below manually? or would it be better to just wait for support to be added?